### ISSUE TITLE
accessibility review of Alerts demo

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#543]
When a demo is implemented showing inline notifications and toast notifications, an a11y review of these working components should be completed to catch any potential issues related to accessibility.

As part of this review:

- For alerts that are dynamic (i.e. not present on page load), ensure that the message body of the alert is announced by the screen reader.
- For alerts that are not dynamic, and are visible on page load, ensure that aria-live is not included on the component.

**Comment by jessiehuff** [comment]
I believe that this was done in this [PR](https://github.com/patternfly/patternfly-react/pull/3519), is there anything outstanding from this issue that we should look into @jgiardino?

**Comment by jgiardino** [comment]
Agree, the current alerts look good.
